src/api.js

```diff
@@ -0,0 +1,14 @@
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export function getTodos() {
+  return fetch(`${BASE_URL}/todos`)
+    .then(response => response.json())
+    .then(json => json.data)
+    .then(dotos => dotos.filter(({ id, title }) => id && title));
```

Comment:
поспішила тут)
```suggestion
    .then(todos => todos.filter(({ id, title }) => id && title));
```
Круто що вирішила позбутися бракованих об'єктів)